docs/src/components/AppDrawer.js

```diff
@@ -103,17 +113,21 @@ export default class AppDrawer extends Component {
         docked={this.props.docked}
       >
         <div className={classes.nav}>
-          <Toolbar>
+          <Toolbar className={classes.toolbar}>
             <Link className={classes.title} to="/" onClick={this.props.onRequestClose}>
               <Text type="title" gutterBottom>
                 Material UI
               </Text>
-              {process.env.MATERIAL_UI_VERSION ? (
-                <Text type="caption">
-                  {`(v${process.env.MATERIAL_UI_VERSION})`}
-                </Text>
-              ) : null}
             </Link>
+            {process.env.MATERIAL_UI_VERSION ? (
+              <Text type="caption">
+                (<a
```

Comment:
Why using a `(` here?
> /home/ubuntu/material-ui/docs/src/components/AppDrawer.js
  125:17  error  Expected indentation of 18 space characters but found 16                                                            react/jsx-indent-props
  126:17  error  Expected indentation of 18 space characters but found 16                                                            react/jsx-indent-props
  126:87  error  The closing bracket must be aligned with the line containing the opening tag (expected column 17 on the next line)  react/jsx-closing-bracket-location